examples/inspection/plot_linear_model_coefficient_interpretation.py

```diff
@@ -0,0 +1,491 @@
+"""
+==================================================================
+Common pitfalls in interpretation of coefficients of linear models
+==================================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Linear models describe situations in which the target value is expected to be
+a linear combination of the features (see the :ref:`linear_model` User Guide
+section for a description of a set of linear model methods available in
+scikit-learn).
+Coefficients in multiple linear models represent the relationship between the
+given feature (`X[i]`) and the target (`y`) assuming that all the other
```

Comment:
In our convention X[i] is the ith sample, not the ith feature. I'd suggest either X[:, i], or using ```:math:`X_i` ```

Also we can loose the parenthesis